samples/net/aws_iot/child_image/mcuboot/boards/nrf7002dk_nrf5340_cpuapp.overlay

```diff
@@ -0,0 +1,6 @@
+/ {
```

Comment:
Missing license. Maybe we don't use licenses in overlays for some reason? I don't know..